includes/wf_crm_webform_preprocess.inc

```diff
@@ -478,7 +478,8 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
 
               // Set empty value for field with 'Create only' mode.
               if ($multivaluesCreateMode === self::MULTIVALUE_FIELDSET_MODE_CREATE_ONLY) {
-                $val = '';
+                $defaultVal = wf_crm_aval($element, '#default_value');
+                $val = $defaultVal ? $defaultVal : '';
```

Comment:
I think this can be written more concisely as `$val = wf_crm_aval($element, '#default_value', '');`

Reply:
Good point, updated